modules/ROOT/pages/configure-teams.adoc

```diff
@@ -0,0 +1,126 @@
+= Configure Teams
+
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+Every root organization has a main _parent team_ that appears by default. Every user in your organization is a member of the highest parent team and receives all permissions therein.
+You can create _child teams_ under the parent team and configure those teams to have different sets of permissions.
+After you create child teams and assign permissions, you can add users to each of your teams according to what permissions the users should have.
+Further, you can assign _team maintainers_ who can modify users and permissions within the teams they maintain.
+
+Users who have the Organization Administrator permission can also change the name of a team, move child teams under different parent teams, or delete teams.
+
+== Create a Child Team
+
+If you have the Organization Administrator permission or are a team maintainer, you can create child teams under a parent team.
```

Comment:
- Org Admins of the Root Org can do this anywhere
- Org Admins of Business Groups can do this under the Root Team (at which point they become maintainers of said team)
- Team maintainers can do this for teams they maintain